src/engine/source/base/parseEvent.cpp

```diff
@@ -83,14 +83,14 @@ Event parseOssecEvent(const std::string& event)
         {
             // Agent_ID index is between '[' and ']'
             // As the format goes like: ...:[<Agent_ID>....
-            endIdx = event.find("]", startIdx);
+            endIdx = event.find(']', startIdx);
             uint32_t valueSize = (endIdx - startIdx) - 1;
             const auto agentId {event.substr(startIdx + 1, valueSize)};
             parseEvent->setString(agentId, EVENT_AGENT_ID);
 
             // Agent_Name is between '(' and ')'
             startIdx = endIdx + 2; // As the format goes like: ...] (<Agent_Name>...
-            endIdx = event.find(")", startIdx);
+            endIdx = event.find(')', startIdx);
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-(C-plus-plus)#performance

'find' called with a string literal consisting of a single character; use the more effective overload accepting a character